runtime/src/rpc.rs

```diff
@@ -120,15 +122,16 @@ impl InterBtcParachain {
 
         let parachain_rpc = Self {
             api: Arc::new(api),
-            signer: Arc::new(RwLock::new(signer)),
+            nonce: Arc::new(RwLock::new(0)),
```

Comment:
is this always supposed to be zero? Maybe there is a value that gets mapped to the highest available nonce

Reply:
Usage is always supposed to be through `get_fresh_nonce`, which compares it to the on-chain value, and takes the higher of the two